manifests/service.pp

```diff
@@ -8,26 +8,11 @@
 # @api private
 #
 class puppetwebhook::service {
-  include systemd
 
-  systemd::unit_file { 'puppet_webhook.service':
-    ensure  => 'present',
-    content => epp('puppetwebhook/puppet_webhook.service.epp', {
-        'user'        => $puppetwebhook::webhook_user,
-        'webhook_bin' => $puppetwebhook::binfile,
-        'env_path'    => $puppetwebhook::envfile_path,
-      }
-    ),
-  }
-
-  service { 'puppet_webhook':
-    ensure     => running,
+  service { 'puppet-webhook':
```

Comment:
Might need to find a better way to handle this as the puppet-webhook parent service is a systemd `oneshot` service. It only starts/stops the children services, it doesn't report if those services are running or not.

Reply:
People on IRC suggested using systemd sockets. I've mixed feelings about this because it didn't reliable work in the past, but that could be triggered by very old systemd versions. Maybe we could also define the three services in puppet.